curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-modern-javascript-methods-by-building-football-team-cards/63c620161fc2b49ac340ffc4.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-1
 
 # --description--
 
-In this project, you will build a set of football team cards and learn about nested objects, object destructuring, default parameters, event listeners, and switch statements. All of the HTML and CSS for this project has been provided for you.
+In this project, you will build a set of football team cards and learn about nested objects, object destructuring, default parameters, and event listeners. All of the HTML and CSS for this project has been provided for you.
```

Comment:
```suggestion
In this project, you will build a set of football team cards and learn about nested objects, object destructuring, and default parameters. All of the HTML and CSS for this project has been provided for you.
```

Campers have already learned about event listeners. So to be consistent with what we are removing, I think that should go as well.